python/lib/db/query/config.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import select
+from sqlalchemy.orm import Session as Database
+from lib.db.model.config import DbConfig
+from lib.db.model.config_setting import DbConfigSetting
+
+
+def get_config_with_setting_name(db: Database, name: str):
+    """
+    Get a single configuration entry from the database using its configuration setting name, or
+    raise an exception if no entry or several entries are found.
+    """
+
+    return db.execute(select(DbConfig)
+        .join(DbConfig.setting)
+        .where(DbConfigSetting.name == name)
+    ).scalar_one()
```

Comment:
I think it is possible that a config is not set in which case this would return None.

Reply:
I guess that if the config is not existent it will raise an exception but do we catch those? What is the error message when that happens?

Reply:
For now it raises an exception, but in the future I was planning to have a `python/lib/config.py` module to make the queries and handle the exceptions by exiting the program.

Reply:
I think some Config could be not set (like the S3 configs) so probably best to support those cases.